spec/sidekiq_unique_jobs/orphans/ruby_reaper_spec.rb

```diff
@@ -135,5 +135,28 @@
         expect(service).not_to have_received(:orphans)
       end
     end
+
+    context "when a lock is until_expired" do
+      let(:lock_info) do
+        {
+          "job_id" => job_id,
+          "limit" => 1,
+          "lock" => :until_expired,
+          "time" => now_f,
+          "timeout" => nil,
+          "ttl" => 1,
+          "lock_args" => [],
+          "worker" => "MyUniqueJob",
+        }
+      end
+
+      it "clears the lock" do
+        expect(redis { |conn| conn.zcard(SidekiqUniqueJobs::EXPIRING_DIGESTS) }).to eq 1
+        sleep 2
```

Comment:
This spec passes for me locally, but I don't think a sleep this long is really appropriate even if it did work consistently. It's also not really the right place for a spec which is curious about the interaction between both the reaper and batch delete. It's here to demonstrate the issue, but I'd love your thoughts about a better location or tactic for testing this